lib/Driver/Compilation.cpp

```diff
@@ -36,12 +37,14 @@
 #include "llvm/Support/Compiler.h"
 #include "llvm/Support/FileSystem.h"
 #include "llvm/Support/Path.h"
-#include "llvm/Support/raw_ostream.h"
 #include "llvm/Support/Timer.h"
 #include "llvm/Support/YAMLParser.h"
+#include "llvm/Support/raw_ostream.h"
 
 #include "CompilationRecord.h"
 
+#define DEBUG_TYPE "BATCH_MODE"
```

Comment:
Nitpick: it looks like we're not consistent about how we use this macro ("lower-kebab-case"? "UpperCamelCase"? "Title Case"? "Regular sentence case"?), but "UPPER_SNAKE_CASE" is not one of the ones we've used so far. "lower-kebab-case" looks like it makes the most sense, since it's going to be used on the command-line as `-Xllvm -debug-only=foo-bar`.

Reply:
Thanks! fixed. Never heard of "kebab-case" before, but it's a wonderful term. If languages only allowed blanks in identifiers we wouldn't need three ways--but would have other problems. (REX (or was it REXX)?) not only allowed blanks in IDs, but supplied a value for an uninitialized variable: the name of the variable as a string! It was great for small scripts, batch-processed from card decks, where one forgot to quote a string. I never used it, though, and am not advocating such.